### ISSUE TITLE
Add the hello video (re-edit)

### ISSUE THREAD

**Issue by mirisuzanne** [oddbird/oddleventy#537]
https://www.dropbox.com/s/4lr0pqqc20eb0ee/mia%20workshop%20intro%20v2.mov?dl=0

**Comment by mirisuzanne** [comment]
Looking at this, I think we might want to re-record the video - adjusting the script to focus more on teams/companies as a target audience. I did a quick draft based on the original script, but figure Pland might also have some input that changes what we want to say. So I won't record until next week. happy for thoughts on the draft: 

```
I'm Miriam Suzanne. Here at OddBird, we've been helping clients of all sizes build more resilient and maintainable web software and design systems.

Over the last few years, I've also been part of the CSS Working Group, helping develop new CSS features like container queries, cascade layers, scope, and nesting.

We've seen an explosion of these new features in CSS lately, shipping quickly across all the browsers – including several features we thought were impossible. But a lot of companies and teams are missing out – stuck using outdated work-arounds, when there are more efficient approaches available. 

I get it. Changing your approach can be overwhelming. 
- Which features deserve your attention? 
- And where are the hidden traps along the way?

That's why we developed this workshop. It's a series of practical sessions on writing more resilient styles for the web – from code organization to layout, custom properties, and the cascade. 

At each step, we'll pull the language apart to really understand how it works. Then we'll use that understanding to develop more resilient and delightful experiences that hold up across browsers, languages, and device interfaces.

This workshop is for anyone who writes CSS at work. And for teams who want to improve their approach, for writing and maintaining resilient CSS at any scale.

I really believe understanding CSS can make your job a bit less stressful, and your web sites more reliable. 

If you or your company are interested, please reach out. We'd love to help.
```

**Comment by SondraE** [comment]
@mirisuzanne It'll be interesting to hear what Pland says. 

The things I'd change just from my initial reading are the first and second sentences. Bring the first sentence more inline with the audience we've said we'd like to reach. And move your statement explaining what/who/why this workshop to the beginning.

"Here at OddBird we enjoy working with teams pursuing large-scale projects to build more resilient and maintainable web applications and design systems. 

Now we've created a workshop for individuals and teams who want to improve their approach for writing and maintaining resilient CSS at any scale."

**Comment by dvdherron** [comment]
I agree with Sondra that it could be good to move who this is for and why a bit earlier in the script. 

Something like this?

I'm Miriam Suzanne. **I'm one of the founders of OddBird, where we work with teams pursuing large-scale projects to build more resilient and maintainable web applications and design systems.**

Over the last few years, I've also been part of the CSS Working Group, helping develop new CSS features like container queries, cascade layers, scope, and nesting.

**Now, the OddBird team and I have created a workshop for individuals and teams who want to take advantage of these features to improve their approach to writing and maintaining resilient CSS at any scale.**

> > We've seen an explosion of these new features in CSS lately, shipping quickly across all the browsers – including several features we thought were impossible. But a lot of companies and teams are missing out – stuck using outdated work-arounds, when there are more efficient approaches available.
> > I get it. Changing your approach can be overwhelming.
> > 
> > * Which features deserve your attention?
> > * And where are the hidden traps along the way?
> > 
> > That's why we developed this workshop. It's a series of practical sessions on writing more resilient styles for the web – from code organization to layout, custom properties, and the cascade.
> > At each step, we'll pull the language apart to really understand how it works. Then we'll use that understanding to develop more resilient and delightful experiences that hold up across browsers, languages, and device interfaces.

 **This workshop is for anyone who writes CSS at work. And for teams who want to improve their process for writing and maintaining CSS.**
> > I really believe understanding CSS can make your job a bit less stressful, and your web sites more reliable.
> > If you or your company are interested, please reach out. We'd love to help.

**Comment by mirisuzanne** [comment]
Those adjustments look good. If only we had a tool for collaborative text editing… 😏

**Comment by dvdherron** [comment]
I was drawn more to take two as well. You definitely establish why you should be teaching this workshop. 

The audio did seem a little thin and low. And I also think adding in more of your sense of humor could help even out the tone. 

My two cents on the camera angle: Is it possible to adjust the camera so that the shelf in the background lines up with the edge of the frame? It looks like it's at a slight angle. 

As far as content, maybe it's possible to be more direct in the "That's why we made this workshop" section. It could a be a quick list  that incorporates some of the benefits Sondra listed above instead of "writing more resilient styles for the web." 

Otherwise, I think this edit is definitely on the right track.
